src/firefly/js/visualize/HiPSListUtil.js

```diff
@@ -212,107 +97,25 @@ export function resolveHiPSIvoURL(ivoOrUrl) {
     if (ivoOrUrl.startsWith('http')) return Promise.resolve(ivoOrUrl);
     if (isBlankHiPSURL(ivoOrUrl)) ivoOrUrl= BLANK_HIPS_URL;
 
-    return getHiPSSurveysTable([HiPSDataType.image, HiPSDataType.cube], 'hipsResolveTable')
-        .then( (tableModel) => {
-            const ivoIdx= getColumnIdx(tableModel, IVO_ID_COL);
-            const urlIdx= getColumnIdx(tableModel, URL_COL);
-            if (ivoIdx<0 || urlIdx<1) return undefined;
-            const lowerIvo= ivoOrUrl.toLowerCase();
-            // now match the table
-            const foundRow= get(tableModel,'tableData.data', []).find( (row) =>
-                                row[ivoIdx] && row[ivoIdx].toLowerCase().includes(lowerIvo) );
-            const replaceUrl= foundRow && foundRow[urlIdx];
-            return replaceUrl || ivoOrUrl;
-        });
-}
-
-
-function stripTrailingSlash(url) {
-    if (typeof url === 'string') {
-        return url.trim().replace(/\/$/, '');
-    }
-    return url;
-}
-
-/**
- *
- * @param tbl table
- * @param cellVal url or ivoId
- * @param columnName  URL_COL or IVO_ID_COL
- * @returns {*}
- */
-export function indexInHiPSSurveys(tbl, cellVal, columnName = URL_COL) {
-    if (!cellVal || typeof cellVal !== 'string') return 0;
-    const cIdx = tbl ? getColumnIdx(tbl, columnName) : -1;
-
-    if (tbl && cIdx >= 0) {
-        const newVal = columnName === URL_COL ? stripTrailingSlash(cellVal).toLowerCase() : cellVal.toLowerCase();
-
-        const data = get(tbl, ['tableData', 'data']);
-
-        return data.findIndex((s) => {
-            return columnName === URL_COL ? (stripTrailingSlash(s[cIdx]).toLowerCase() === newVal):
-                   s[cIdx].toLowerCase().includes(newVal);
+    const findInTable = (tableModel) => {
+        const ivoIdx= getColumnIdx(tableModel, IVO_ID_COL);
+        const urlIdx= getColumnIdx(tableModel, URL_COL);
+        if (ivoIdx<0 || urlIdx<1) return undefined;
+        const lowerIvo= ivoOrUrl.toLowerCase();
+        // now match the table
+        const foundRow= get(tableModel,'tableData.data', []).find( (row) =>
+            row?.[ivoIdx]?.toLowerCase().includes(lowerIvo) );
+        return foundRow?.[urlIdx] || ivoOrUrl;
+    };
+
+    if (FULL_HIPS_TABLE) {
+        return Promise.resolve(findInTable());
```

Comment:
should be:
```
 return Promise.resolve(findInTable(FULL_HIPS_TABLE));
```

Reply:
Thanks, it's fixed.  Test instance updated.